backend/buildeR.BLL/Services/SynchronizationService.cs

```diff
@@ -77,13 +77,17 @@ public async Task<bool> CheckIfUserHasToken(int userId)
         }
         public async Task RegisterWebhook(int projectId, string callback)
         {
-            callback += $"/{projectId}/github";
+            callback = $"http://5451f9723672.ngrok.io/webhooks/{projectId}/github";
```

Comment:
this won't work on PROD env

Reply:
thank you, I missed to remove mocked data